File: CursoJava/Exer/src/Outros/ExerChat/ExerciciosDeCondicional/Exer16.java
/*
16. Escreva um programa que determine o maior de dois números.
 */

package Outros.ExerChat.ExerciciosDeCondicional;

import java.util.Scanner;

public class Exer16 {
    public static void main(String[] args) {
        int num1, num2;

        Scanner sc = new Scanner(System.in);

        System.out.println("Informe o primeiro valor: ");
        num1 = sc.nextInt();
        System.out.println("Informe o segundo valor: ");
        num2 = sc.nextInt();

        if (num1 > num2) {
            System.out.println("O maior número é o " + num1);
        }
        else {
            System.out.println("O maior número é o " + num2);
        }
    }
}

File: CursoLuizOtavio/CursoPython3/exer/lista_64_Exer/exer16.py
"""
16) Escreva um programa que leia dois números e exiba mensagem informando o valor do maior número e o valor do 
menor número. Se os dois números forem iguais, o programa deve exibir mensagem informando este fato. 
"""

num1 = float(input('Digite o primeiro número: '))
num2 = float(input('Digite o segundo número: '))

if (num1 == num2):
    print('Os números são iguais')
elif (num1 > num2):
    print(f'O maior número é {num1}')
    print(f'O menor número é {num2}')
else:
    print(f'O maior número é {num2}')
    print(f'O menor número é {num1}')
